packages/ui/src/theme/css/component/_fieldControl.scss

```diff
@@ -71,16 +71,6 @@
     }
   }
 
-  // for disabled styles
```

Comment:
These are moved further down the file so that they can take precedence over the `quiet` variation css. Fixes https://github.com/aws-amplify/amplify-ui/issues/4409